code/modules/awaymissions/zlevel.dm

```diff
@@ -62,6 +62,13 @@ var/global/list/potentialRandomZlevels = generateMapList(filename = "config/away
 /proc/seedRuins(list/z_levels = null, budget = 0, whitelist = /area/space, list/potentialRuins = space_ruins_templates)
 	if(!z_levels || !z_levels.len)
 		z_levels = list(1)

```

Comment:
If you provide no z_levels, generate on Z level 1? Seems like z_levels == null should just be an error, rather then have this un-intuitive behaviour